src/wp-includes/blocks/latest-posts.php

```diff
@@ -152,10 +152,9 @@ function render_block_core_latest_posts( $attributes ) {
 				if ( $excerpt_length <= $block_core_latest_posts_excerpt_length ) {
 					$trimmed_excerpt  = substr( $trimmed_excerpt, 0, -11 );
 					$trimmed_excerpt .= sprintf(
-						/* translators: 1: A URL to a post, 2: The static string "Read more", 3: The post title only visible to screen readers. */
-						__( '… <a href="%1$s" rel="noopener noreferrer">%2$s<span class="screen-reader-text">: %3$s</span></a>' ),
+						/* translators: The static string "Read more". %1$s: A URL to a post, %2$s: Hidden accessibility text: Post title */
+						__( '… <a href="%1$s" rel="noopener noreferrer">Read more<span class="screen-reader-text">: %2$s</span></a>' ),
```

Comment:
Curious why we're using "Read more: {title}" here, as it reads a bit weird. Most existing core themes have been using something like "Continue reading {title}", which makes for a better reading flow.

Also see https://github.com/WordPress/wordpress-develop/pull/5439#discussion_r1350597442

Reply:
I'm not sure why this wording has been chosen, so pinging @ramonjd @andrewserong to hopefully help here.

Reply:
Thanks for the ping. "Read more" has been a feature of this block for a long time and I didn't want to change things up too much. So the basic reason was backwards compatibility.

That meant deciding how to structure the string so that it made sense grammatically. A semi-colon is fine.

The other option could have been "Read more _of_", but I discarded the superfluous preposition. 

I'm happy for the wording to change completely if folks think that it best. As mentioned my only intention was to avoid the situation where existing sites installing 6.4 and are left wondering why the "Read more" link text has changed.

P.S.: I'll make a note to backport any changes made here back to Gutenberg. Thanks a lot for helping to tidy this up, everyone!

Reply:
Yeah the main reason the post title is added after the read more is so that when a screen reader provides a list of links on the page there is some context to each link and not just a bunch of "read more"s. In those circumstances something like "continue reading" won't make any more sense than "read more". 

In any case this is probably not something to be changed during Beta as it's not, strictly speaking, a bug 😄